test/testMultiDefinition/staticTestLib.cpp

```diff
@@ -0,0 +1,14 @@
+#include "edlib.hpp"
+
+/***
```

Comment:
How about we instead create README.md in testMultiDefinition/ directory and explain what is this about there?
And you can say: 

```
In this test, we want to ensure that when edlib is included in multiple complication units, no "multiple definitions" error will occur.
This error is something we are concerned with because due to using templates we have function definitions in header, and they cause this kind of error in this kind of situation. Solution is to make functions "inlined", and this test ensures that.
```

Also, pls add to this new README instructions how to run this test.
Later, we will make it a part of CI / have it run automatically -> actually, let's put a TODO for that (also in this new README).

Btw, you checked that this test fails when an inline is missing, right? Sorry for asking this, you probably did, but just to be sure.

Reply:
Yes, I ran the test and add inline specifier based on errors like below:
`staticTestLib.cpp:(.text+0x0): multiple definition of edlib::internal::obtainAlignmentTraceback(int, int, int, edlib::internal::AlignmentData const*, unsigned char**, int*)'`